src/components/SingleArticle/SingleArticle.tsx

```diff
@@ -28,6 +30,28 @@ export const SingleArticle = ({ article }: { article: ArticleRecord }) => {
   })
   const publishedDateObj = new Date(publishedDate)
 
+  const componentBlockRenderers: InferRenderersForComponentBlocks<
```

Comment:
Question: is it possible to add a unit test for this function?

Reply:
Not sure? I'll dig into that and see what I come up with